lib/dice.py

```diff
@@ -0,0 +1,9 @@
+from random import randint
+
+    def roll(self, number, sides, modifier):
```

Comment:
Function shouldn't be indented since it's "under" anything

Reply:
sorry, it used to be, fixed